build_library/grub.cfg

```diff
@@ -27,9 +27,31 @@ if [ "$grub_platform" = "efi" ]; then
    getenv -e SetupMode -g 8be4df61-93ca-11d2-aa0d-00e098032b8c -b setupmode
    if [ "$sb" = "01" -a "$setupmode" = "00" ]; then
       set secure_boot="1"
+      getenv -e gpg -g b8ade7d5-d400-4213-8d15-d47be0a621bf -b gpg
```

Comment:
Seems like the name in UEFI should be more descriptive.

Also should check that a value was found before trusting it.